### DISCUSSION TITLE
Region dla Azure DC Warszawa - data

### DISCUSSION THREAD

**Discussion by bryn1u** [AzureCommunityPL/group#177] [Ogólna]
Dzień dobry,
Czy orientuję się ktoś do kiedy ma powstać DC w Warszawie jako nowy region dla Azure ? Szukam jakichs dat, ale nic kompletnie nie moge znalezc. Znalazłem jedynie, że teoretycznie 2 lata od zaczęcia prac, ale dużo to nie mowi. Ma ktos jakieś bardziej szczegółowe info ?

Dzieki z góry,

**Comment by eklime** [comment]
Oficjalnie nikt Ci tu nie odpowie bo to ściśle tajne ;)

**Reply by bryn1u** [reply]
Tak to prawda. Z tego co się dowiedziałem to 2022 H1. Teoretycznie 2 lata od zaczęcia budowy. A ile w tym prawdy jest to sie okaże.

**Reply by eklime** [reply]
Można szacować na podstawie danych historycznych i prześledzić newsy z ogłoszenia i GA np. Szwajcarii czy Francji ;)
